### PR TITLE
fix(@toss/utils): ceilUnit parameter was missing

### PR DESCRIPTION
**Pull request by cha2hyun**
## Overview

Only floorToUnit was provided. 

## PR Checklist

- [✅] I read and included theses actions below

1. I have read the [Contributing Guide](https://github.com/toss/slash/blob/main/.github/CONTRIBUTING.md)
2. I have written documents and tests, if needed.

### CODE REVIEW

File: packages/common/utils/src/Numbers.ts [hunk 1]
```diff
@@ -34,14 +34,22 @@ export const floorToUnit = createNumberFormatterBy(Math.floor);
...
+  if (options.floorUnit > 0) {
+    val = floorToUnit(value, options.floorUnit || 1);
+  } else if (options.ceilUnit > 0) {
+    val = ceilToUnit(value, options.ceilUnit || 1);
+  }
 
-  if (flooredVal === 0) {
-    return '0';
+  if (val === 0) {
+    return "0";
```
Comment (po4tion) [inline packages/common/utils/src/Numbers.ts:49]: "slash" uses "singlequote".

You can refer this prettierrc file.

[ .prettierrc]
image

**Review by po4tion — CHANGES_REQUESTED** [review]

File: packages/common/utils/src/Numbers.ts (hunk 1 above)
Comment (cha2hyun) [reply packages/common/utils/src/Numbers.ts]: Thanks for reviewing. I changed it.

**Comment by cha2hyun** [comment]
@raon0211 
Thank you for your review. 🚀

The document already had the ceilUnit parameter written on it, but it was not actually implemented. 
This "PR" corrected the part that did not work.

https://slash.page/libraries/common/utils/src/Numbers_formatToKRW.i18n
image

**Review by raon0211 — APPROVED** [review]
Thanks!

**Pull request merged**